src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -281,23 +281,27 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
         case gjre: GoogleJsonResponseException if gjre.getDetails.getCode == StatusCodes.Conflict.intValue => None
       }
       _ <- creationOperationId match {
-        case Some(opId) => pollShellProjectCreation(opId) //poll until it's created
+        case Some(opId) => for {
+          _ <- pollShellProjectCreation(opId) //poll until it's created
+          enableBillingOperationId <- googleServiceManagerDAO.enableService(projectName, "storage-api.googleapis.com")
```

Comment:
This is not enable billing operation. Does enabling this api require billing?

Reply:
no it doesn't require billing to be enabled. i was trying to get this out quick before leaving while also thinking of something else and the two things merged in my head somehow. so anything that says "enable billing" should be "enable storage apis". 

something along the lines of "do what i mean, not what i say" or whatever that saying is...